### PR TITLE
feat : added playlist page

### PR DESCRIPTION
**Pull request by rohit8686**
Features : 

- Create Playlists
- Add videos to playlist 
- Delete playlists
- Delete videos from playlist

### CODE REVIEW

File: src/frontend/components/PlaylistVideoOptions/PlaylistVideoOptions.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <>
+      <div className="video-options">
+        <button
+          className="video-options-btn"
+          onClick={() => removeVideoFromPlaylist(playlistId, videoId)}
+        >
+          Delete video from playlist
+        </button>
+      </div>
+    </>
```
Comment (romabulani) [inline src/frontend/components/PlaylistVideoOptions/PlaylistVideoOptions.jsx:18]: You can remove the Fragment here if you are not going to add other elements

**Review by romabulani — APPROVED** [review]
- Clean and Modular code
- Functionality is working as expected
- Good to merge

**Comment by anandsr21harshit** [comment]
Clean and Modular code
Functionality is working as expected
You can merge it

**Review by anandsr21harshit — APPROVED** [review]

File: src/frontend/Routes/Routing.jsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { Home } from "../pages/Home/Home";
 import { Liked } from "../pages/Liked/Liked";
 import { Login } from "../pages/Login/Login";
 import { Playlist } from "../pages/Playlist/Playlist";
+import { PlaylistItems } from "../pages/PlaylistItems/PlaylistItems";
 import { Signup } from "../pages/Signup/Signup";
 import { Video } from "../pages/Video/Video";
 import { Videos } from "../pages/Videos/Videos";
```
Comment (anandsr21harshit) [inline src/frontend/Routes/Routing.jsx:11]: you can use barrel import

**Pull request merged**